packages/files/src/create-directory.ts

```diff
@@ -25,7 +26,7 @@ import type { FilePath } from './types.js';
  * ```
  */
 export function createDirectory(filePath: FilePath): void {
-  const directory = dirname(filePath);
+  const directory = dirname(normalizeFilePath(filePath));
```

Comment:
Consider using `normalizeFilePath()` in `@codemod-utils/files` in the absence of `join()`.